docs/format.rst

```diff
@@ -482,6 +485,14 @@ Exceptions
    exceptions. See {tagname} of Proposal section for more detail about the
    conversion.
 
+.. NOT REVIEWED YET
+
+#. "name" part of {tagfield} in a tag generated by Universal-ctags may
+   contain numerical characters. Universal-ctags doesn't allow users
+   to use a numerical character as the initial letter of the "name"
+   when defining a kind.  However, it allows the users to use
+   numerical characters in the rest letters.
```

Comment:
I suggest re-wording this, something like:

```
#. "name" part of {tagfield} in a tag generated by Universal-ctags may
   contain numeric characters, but the first letter of the "name" must
   be a letter.

Reply:
hmm ... I see that letter is a bit overloaded to mean character in existing error messages ...

Reply:
... so perhaps:

```
#. "name" part of {tagfield} in a tag generated by Universal-ctags may
   contain numeric characters, but the first character of the "name"
   must be alphabetic.
```

Reply:
(Reflected to the local source tree).